ranker/labelled_ranker.py

```diff
@@ -0,0 +1,118 @@
+import json, requests
+import simplejson
+import json
+
+filters=['easy','medium','hard','bug']
+stats=['additions','deletions','commits']
+scoring=dict(zip(filters+stats,[20,50,100,10]+[1,1,5]))
+users={}
+def init_user(login):
+    users[login]=dict(zip(filters+stats,len(filters+stats)*[0]))
+
+def req(get,params={}):
+    """Function to send HTTP Requests"""
+    params['access_token']='d0fec873c0a68b8e8b29b2403914a41f59479dc5'
+    api_host="https://api.github.com"
+    url=api_host+get
+    resp = requests.get(url=url, params=params)
+
+    try:
+        data = json.loads(resp.text)
+    except:
+        data=[]
+    return data
+
+
+def get_repos(org):
+    """Returns Repos under an organization"""
+    url='/orgs/'+org+'/repos'
+    repo_list=req(url)
+    repos=[]
+    for repo in repo_list:
+        repos.append(repo["name"])
+    return repos
+
+def get_repo_issues(repo,org,closed=True):
+    """Returns a list of issues(closed issues, by default) under a repository"""
+    url='/repos/'+org+'/'+repo+'/issues'
+    params={'state':'closed'} if closed else {}
+    issues=req(url,params)
+    return issues
+
+
+
+def filter_issues(issue_list):
+    """Returns a list of issues which have specific labels"""
+
```

Comment:
Make it consistent, if you are having a new line after doc string then make it for all, otherwise just don't add it. When in doubts always use `pep8`.